### ISSUE TITLE
On every change, we replace the entire todo collection instead of only updating what changed

### ISSUE THREAD

**Issue by IlyaKriel** [uNople/Todo#8]
todo.get() 
Overwriting the code each time without checking if there were actually any changes.
You mentioned it would be a problem in the future.

**Comment by uNople** [comment]
Yep, good idea. We should fix this sooner rather than later, for sure

**Comment by uNople** [comment]
![image](https://user-images.githubusercontent.com/2319718/108579320-37dfe300-738b-11eb-8ee8-f14d63c60808.png)
